packages/@coorpacademy-components/src/molecule/learner-skill-card/index.js

```diff
@@ -49,10 +44,12 @@ const LearnerSkillCard = (props, context) => {
 
   const buttonExploreProps = {
```

Comment:
can you do the same thing for the learning-priority-card? It uses the same old logic

I merged the pull request, so you maybe need to rebase,

thanks

Reply:
Thanks Adem, I've updated for it too!